docs-client/src/containers/MethodPage/DebugPage.tsx

```diff
@@ -214,9 +219,10 @@ const DebugPage: React.FunctionComponent<Props> = ({
     setRequestBody(urlRequestBody || method.exampleRequests[0] || '');
     setAdditionalPath(urlPath || '');
     setAdditionalQueries(urlQueries || '');
-    setDebugFormIsOpen(
-      (isOpen) => isOpen || urlRequestBody !== '' || urlQueries !== '',
-    );
```

Comment:
I guess this problem is fixed by #5072 since these lines are removed at https://github.com/line/armeria/pull/5072/files#diff-812b896083a854df7bc9731f6b8e0364ee398948b69150196d4d87268a73bc03L217

Reply:
Yes, it seems like this PR became an optional functionality that allows users to share URLs that automatically open up the debug form. What do you think?

Reply:
That sounds good to open up the debug modal when a user intended it.